packages/client/src/StreamRegistry.ts

```diff
@@ -103,19 +103,19 @@ export class StreamRegistry implements Context {
         throw new NotFoundError('Stream: id=' + id)
     }
 
-    async hasPermission(streamId: string, userAddess: EthereumAddress, operation: StreamOperation) {
+    async hasPermission(streamId: string, userAddess: EthereumAddress, permission: StreamPermission) {
         return this.streamRegistryContractReadonly.hasPermission(streamId, userAddess,
-            StreamRegistry.streamOperationToSolidityType(operation))
+            StreamRegistry.StreamPermissionToSolidityType(permission))
```

Comment:
`StreamPermissionToSolidityType` should probably be `streamPermissionToSolidityType`, likely find/replace did this

Reply:
Yeah, fixed